packages/devtools_app/lib/src/shared/version.dart

```diff
@@ -42,15 +42,15 @@ class FlutterVersion extends SemanticVersion {
 
   final String channel;
 
-  final String repositoryUrl;
+  final String? repositoryUrl;
```

Comment:
why is repositoryUrl the only member that is nullable here? I would expect that they are either all nullable or none since we are just parsing json?

Reply:
fixed